exe/fcrepo_wrapper

```diff
@@ -8,7 +8,7 @@ options = {}
 OptionParser.new do |opts|
   opts.banner = "Usage: fcrepo_wrapper [options]"
 
-  opts.on("--config", "Load configuration from a file") do |v|
+  opts.on("-cFILE",  "--config FILE", "Load configuration from a file") do |v|
```

Comment:
I think the short-name `-c` is used over in `solr_wrapper` for some other function. I don't know if we should be worried about introducing that kind of inconsistency between the two project.